pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyGovernanceDao.java

```diff
@@ -99,6 +100,7 @@ default int removeAgeUpCandidates(Set<Long> candidateIds) {
     @UseStringTemplateSqlLocator
     @SqlQuery("queryAllAgeUpCandidatesByStudyId")
     @RegisterConstructorMapper(AgeUpCandidate.class)
+    @FetchSize(300)
```

Comment:
Just flagging this since it wasn't mentioned in the overview.  Seems like a nice incidental improvement!